### PR TITLE
Add template read/write/test file

### PR DESCRIPTION
**Pull request by AJ-Owen**
Adds a generic starter.py file that can be copied into each months challenge folder and updated to fit specific challenges.

Methods to update:
    parse_input(): Fit to problem input
    write_output(): Ouput results in correct format
    calculate_result(): Implement algorithm
    main(): Update filenames for different datasets

### CODE REVIEW

File: generic_starter.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        num_test_cases = int(f.readline())
+        test_cases = []
+        for _ in range(num_test_cases):
+            test_cases.append(f.readline())
+    return test_cases
+
+
+def write_output(filename: str, values: list) -> None:
+    """
+    Writes the data to file in correct format for KickStart 2020 round B
```
Comment (elichad) [inline generic_starter.py]: This block comment is referencing a specific challenge – could you generalize it?

I think a few of the other PRs also need this comment changing, if you could double check those too.

**Review by elichad — CHANGES_REQUESTED** [review]
This looks great, thanks for making the template. Only thing that needs changing is a comment, otherwise I'm happy

File: generic_starter.py (hunk 1 above)
Comment (AJ-Owen) [reply generic_starter.py]: Comment has been updated

File: generic_starter.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        for _ in range(num_test_cases):
+            test_cases.append(f.readline())
+    return test_cases
+
+
+def write_output(filename: str, values: list) -> None:
+    """
+    Writes the data to file in a generic format for Google Kickstarter
+    filename str: Name of file to write output to
+    values list[tuple(int, int)]: Final rover positions
```
Comment (elichad) [inline generic_starter.py]: I meant this line should be generalised too, should have put the original comment here really

**Review by elichad — APPROVED** [review]

**Pull request merged**